src/Adapter/OpenID.php

```diff
@@ -82,7 +82,7 @@ public function authenticate()
             return true;
         }
 
-        $openid_mode = filter_input(INPUT_GET, 'openid_mode');
+        $openid_mode = filter_input($_SERVER['REQUEST_METHOD'] === 'POST' ? INPUT_POST : INPUT_GET, 'openid_mode');
```

Comment:
what if use `$_REQUEST` superglobal array which contains GET + POST?

```php
if (!empty($_REQUEST['openid_mode'])) {
    $this->authenticateBegin();
}
else {
    ...
}
```

Reply:
I think you meant `empty($_REQUEST['openid_mode'])`
At all without difference, but I just left `filter_input` as it was

Reply:
yeah, 💯  (updated)

can you update this PR according to my comment?